### PR TITLE
change branch name for created PRs in workflow

### PR DESCRIPTION
**Pull request by rosalieper**

### CODE REVIEW

File: .github/workflows/docker.build.yml [hunk 1]
```diff
@@ -70,7 +70,7 @@ jobs:
         with:
           commit-message: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
           title: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
-          branch: deploy-to-staging-${{ steps.docker_meta.outputs.tags }}
+          branch: deploy-to-staging
```
Comment (m90) [inline .github/workflows/docker.build.yml:73]: I think these need to be unique still, else we'll create conflicts in the `wbaas-deploy` repo when people do not instantly merge these PRs. Maybe we can find something else that is unique, but doesn't contain colons (which is why this broke). Maybe we can use `GITHUB_RUN_ID` instead which is a unique identifier for each workflow run: https://docs.github.com/en/actions/learn-github-actions/variables

Comment (rosalieper) [reply .github/workflows/docker.build.yml:73]: that is a great idea. thanks

File: .github/workflows/docker.build.yml [hunk 2]
```diff
@@ -70,7 +70,7 @@ jobs:
         with:
           commit-message: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
           title: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
-          branch: deploy-to-staging-${{ steps.docker_meta.outputs.tags }}
+          branch: deploy-to-staging-${{GITHUB_RUN_ID}}
```
Comment (m90) [inline .github/workflows/docker.build.yml:73]: I'm not entirely sure how to access env vars in action parameters. I can't find anything in the official docs right now, but random examples from Googling are all using `${{ env.GITHUB_RUN_ID }}`

Comment (rosalieper) [reply .github/workflows/docker.build.yml:73]: ok

Comment (m90) [reply .github/workflows/docker.build.yml]: Ah, yeah looks like this should work https://docs.github.com/en/actions/learn-github-actions/contexts#example-usage-of-the-env-context

**Review by m90 — APPROVED** [review]
Let's give this another try :crossed_fingers:

**Pull request merged**